<?xml version="1.0"?>
<launch>

    <include file="$(find bitbots_ros_control)/launch/ros_control.launch" />

    <include file="$(find bitbots_bringup)/launch/load_robot_description.launch" />

    <rosparam file="$(find bitbots_quintic_walk)/config/walking_wolfgang_robot.yaml" command="load"/>

    <node name="walking" pkg="bitbots_quintic_walk" type="WalkNode" output="screen">
        <remap from="/walking_motor_goals" to="/DynamixelController/command"/>
    </node>


</launch>